tests/python/gaia-ui-tests/gaiatest/apps/search/regions/browser.py

```diff
@@ -47,8 +48,8 @@ def tap_menu_button(self):
         Wait(self.marionette).until(
             expected.element_displayed(*self._browser_menu_locator))
 
-    def tap_add_to_home(self):
-        element = self._root_element.find_element(*self._add_to_home_button_locator)
+    def tap_pin_button(self):
+        element = self._root_element.find_element(*self._pin_button_locator)
         Wait(self.marionette).until(expected.element_displayed(element))
         element.tap()
         from gaiatest.apps.homescreen.regions.bookmark_menu import BookmarkMenu
```

Comment:
BookmarkMenu is an odd name now, because it's called pinning now. I guess you should rename this to PinMenu or something.